### PR TITLE
[PRP-348] Add additional survey links

### PR DESCRIPTION
**Pull request by rocketnova**
## Ticket

https://wicmtdp.atlassian.net/browse/PRP-348

- Add feedback survey link in the footer
- Add post-recertification-appointment survey link on the index page

We want to increase the opportunities for participants to provide feedback.

All the snapshots had to be changed because of the changes to the Layout footer. **Please help me verify that no other changes snuck in.**

### CODE REVIEW

File: participant/app/routes/$localAgency/recertify/index.tsx [hunk 1]
```diff
@@ -39,7 +41,7 @@ export default function Index() {
         {resetSession === "true" && (
           <Alert
             type="warning"
-            headingLevel="h6"
+            headingLevel="h2"
```
Comment (rocketnova) [inline participant/app/routes/$localAgency/recertify/index.tsx:43]: I changed this for the axe accessibility alert about using the next step up in headings. It does not change the appearance visually.

Comment (microwavenby) [reply participant/app/routes/$localAgency/recertify/index.tsx:43]: Axe has feelings!

File: participant/app/routes/$localAgency/recertify/index.tsx [hunk 2]
```diff
@@ -27,6 +28,7 @@ export const loader: LoaderFunction = async ({
     { headers: headers }
   );
 };
+const feedbackAlertHeading = <Trans i18nKey="Index.feedback.heading" />;
```
Comment (microwavenby) [inline participant/app/routes/$localAgency/recertify/index.tsx]: Could just set this in the prop? 🤷🏻‍♀️

**Review by microwavenby — APPROVED** [review]
🚢 :shipit:

File: participant/app/routes/$localAgency/recertify/index.tsx (hunk 2 above)
Comment (rocketnova) [reply participant/app/routes/$localAgency/recertify/index.tsx]: Hmmmm I tried this yesterday and it wasn't working, but now it is, so I've incorporated it 🤷 . I must have done something wrong yesterday.

**Pull request merged**